docs/production-deployment/cloud/nexus/use-cases.mdx

```diff
@@ -0,0 +1,83 @@
+Nexus enables each team to have their own Namespace for improved security, troubleshooting, and fault isolation.
+
+Temporal Nexus enables a modular application design that can evolve as you grow.
+Using Nexus Endpoints in a monolithic Namespace decouples callers from handlers, so Nexus Services can be moved to different Namespaces with minimal config changes and no downtime.
+
+**Follow these steps to decompose a large monolithic Namespace:**
+
+1. Hide service implementations behind a Nexus Endpoint.
+   1. Add [Nexus Endpoints](/nexus/registry-endpoints#nexus-endpoints) to the [Nexus Registry](/nexus/registry-endpoints#nexus-registry), with monolithic Namespace as the target.
+   2. Add [Nexus Services](/nexus/services-operations) to the same Workers as the Temporal primitives being abstracted.
+2. Service consumers use the Nexus Endpoint instead of the underlying implementation.
+   1. This can be done incrementally, until there are no direct caller dependencies on the underlying service implementations, that is the underlying Temporal Workflow and Activity primitives.
+3. Move service implementations to a different Namespace.
+   1. Create a new Namespace.
+   2. Add a Worker deployment with the Nexus Service.
+   3. Update the Nexus Endpoint target to the new Namespace.
+   4. Configure the Endpoint allowlist to allow calls from the original monolithic Namespace.
+   5. New Nexus Operations will be routed from callers in the monolithic Namespace to the new Namespace.
+4. Quiesce [Nexus Operations](/nexus/services-operations#nexus-operations) on the previous target Namespace.
+   1. Leave the previous Worker deployment running until all existing Nexus Operations in the previous Namespace have completed.
+      This includes their underlying Workflows, if any.
+   2. Previous Workers in the monolithic Namespace can have the service implementation removed, since it is now being served from the new Endpoint target Namespace.
+
+Please note the following points:
+
+- For long-lived entity Workflows, new Nexus requests will be routed to the new target Namespace.
+  Long-lived entity-Workflow will still be running on the previous Namespace.
+- When using Signal-with-Start behind a Nexus Operation, a new entity-Workflow will start in the new target Namespace.
+- A cancel Nexus Operation request from a caller Workflow will be routed through the Nexus Endpoint to the new target Namespace.
+  The new Namespace may not have the underlying Workflow.
+
+In these scenarios, Temporal can provide guidance on different migration strategies.
+
+## Learn more
+
+<RelatedReadContainer>
+  <RelatedReadItem path="/evaluate/nexus" text="Evaluate Nexus" archetype="evaluate" />
+  <RelatedReadItem path="/cloud/nexus" text="Production deployment in Temporal Cloud" archetype="feature-guide" />
```

Comment:
```suggestion
To learn more, check out the [Evaluate Nexus](/evaluate/nexus) section in our documentation.
```
The second link directs you back to this section so it can be removed.